src/App.js

```diff
@@ -1,13 +1,21 @@
+import SearchResults from './SearchResults';
+import  Footer from './Footer';
+import TouristInfoCards  from './TouristInfoCards';
+import  Heading  from './Heading';
 import React from "react";
 
 import Bookings from "./Bookings";
 import "./App.css";
 
+
 const App = () => {
   return (
     <div className="App">
-      <header className="App-header">CYF Hotel</header>
+      <Heading     />
```

Comment:
```suggestion
      <Heading />
```

you dont need a tab, just a regualar space would be enough

same for 

````
      <TouristInfoCards     />
      <Footer     />
````